exercises/concept/sorting-room/.meta/exemplar.go

```diff
@@ -0,0 +1,55 @@
+package sorting
```

Comment:
In case you are open to more general feedback on the exercise itself, here it is:
Of course the exercise works as is but currently the two boolean tasks don't practice anything that was taught in the concepts. Additionally I thought whether it would be possible to modify the exercise so that the student actually needs to use `x, ok := T(y)` and `strconv`.

A possible way to address both things would be to make the exercise only about number sorting and to remove the boolean parts from the existing content. Now that we "made some space" you could add another task about a very special type of number where the student needs to check for this number type and use strconv to extract the number. Something along the lines of this:
```go
type FancyNumber struct {
	n string
}

func (i FancyNumber) Value() string {
	return i.n
}

type FancyNumberBox interface {
	Value() string
}

func ExtractFancyNumber(nb FancyNumberBox) int {
	fancyNum, ok := nb.(FancyNumber)
	if !ok {
		return 0
	}

	num, _ := strconv.Atoi(fancyNum.Value())
	return num
}
```

Reply:
Yep, that sounds good. Went with this option.